UI/Selenium/Steps/ManageWorkAllocationSteps.cs

```diff
@@ -0,0 +1,138 @@
+using System.Threading;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using TechTalk.SpecFlow;
+using TestFramework;
+using UI.Pages;
+using UI.Utilities;
+using System;
+using RazorEngine.Compilation.ImpromptuInterface.InvokeExt;
+
+namespace UI.Steps;
+
+[Binding]
+public class ManageWorkAllocationSteps : ObjectFactory
+{
+    private readonly ScenarioContext _scenarioContext;
+    SelectYourHearingListSteps selectYourHearingListSteps;
+
+    public ManageWorkAllocationSteps(ScenarioContext scenarioContext, SelectYourHearingListSteps selectYourHearingListSteps)
```

Comment:
and the constructor should be 
```suggestion
    public ManageWorkAllocationSteps(ScenarioContext scenarioContext, SelectYourHearingListSteps selectYourHearingListSteps)
    {
         _scenarioContext = scenarioContext;
         _selectYourHearingListSteps = selectYourHearingListSteps;
     }
```
```

Reply:
Changes apply